server/repository/migrations/postgres/2022-10-11T10-22_v102/up.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TYPE key_type ADD VALUE IF NOT EXISTS 'SETTINGS_DISPLAY_CUSTOM_THEME';
```

Comment:
renamed file, which incorrectly had `100` rather than `10` in the name. To avoid migration issues, have reintroduced the `NOT EXISTS` check